packages/orchestration/src/contracts/localchainAccountHolder.js

```diff
@@ -0,0 +1,154 @@
+// @ts-check
+/** @file Use-object for the owner of a localchain account */
+import { AmountShape } from '@agoric/ertp';
+import { makeTracer } from '@agoric/internal';
+import { UnguardedHelperI } from '@agoric/internal/src/typeGuards.js';
+import { M, prepareExoClassKit } from '@agoric/vat-data';
+import { TopicsRecordShape } from '@agoric/zoe/src/contractSupport/index.js';
+import { E } from '@endo/far';
+
+const trace = makeTracer('LCAH');
+
+const { Fail } = assert;
+/**
+ * @typedef {object} LocalChainAccountNotification
+ * @property {string} address
+ */
+
+/**
+ * @typedef {{
+ *   topicKit: import('@agoric/zoe/src/contractSupport/recorder.js').RecorderKit<LocalChainAccountNotification>;
+ *   account: import('@agoric/vats/src/localchain.js').LocalChainAccount | null;
+ * }} State
+ */
+
+const HolderI = M.interface('holder', {
+  getPublicTopics: M.call().returns(TopicsRecordShape),
+  makeDelegateInvitation: M.call(M.string(), AmountShape).returns(M.promise()),
+  makeCloseAccountInvitation: M.call().returns(M.promise()),
+  makeTransferAccountInvitation: M.call().returns(M.promise()),
+});
+
+/** @type {{ [name: string]: [description: string, valueShape: Pattern] }} */
+const PUBLIC_TOPICS = {
+  account: ['Account holder status', M.any()],
+};
+
+/**
+ * @param {import('@agoric/ertp').Baggage} baggage
+ * @param {import('@agoric/zoe/src/contractSupport/recorder.js').MakeRecorderKit} makeRecorderKit
+ * @param {ZCF} zcf
+ */
+export const prepareAccountHolder = (baggage, makeRecorderKit, zcf) => {
+  const makeAccountHolderKit = prepareExoClassKit(
+    baggage,
+    'Account Holder',
+    {
+      helper: UnguardedHelperI,
+      holder: HolderI,
+      invitationMakers: M.interface('invitationMakers', {
+        Delegate: HolderI.payload.methodGuards.makeDelegateInvitation,
```

Comment:
Not in the original requirements, but additional actions we'll likely need - 
- Ability to claim (and withdraw) rewards
- Ability to undelegate / unbond
- Ability to redelegate to a different validator

Queries are also coming to mind - 
- Current Delegations
- Claimable Rewards
- Current Redelegations 
- Current Unbondings